package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "p6-arqam-adnan-dedhawala",
+  "version": "1.0.0",
+  "description": "Welcome to Pesto assignments.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "format": "npm-run-all -p format:*",
+    "format:code": "npm run prettier:code -- --write",
+    "format:other": "npm run prettier:other -- --write",
+    "prettier:code": "prettier \"**/*.{ts,tsx,js,jsx}\"",
+    "prettier:other": "prettier \"**/*.{css,json,scss,html}\""
```

Comment:
no need of this, also try and add git hooks with husky

Reply:
done